chronicles-rulebook.md

```diff
@@ -2533,15 +2525,15 @@ Item production is the process by which characters are able to craft consumable
 #### Rank V
 
 ##### Oil of Flame
-   >This substance coats a weapon in a flaming oil for the next 5 minutes. During that time, the weapon will do flame type Damage. Applying the oil takes 5 seconds. Example: “5 normal flame”.
+   >This substance coats a weapon in a flaming oil for the next 5 minutes. During that time, the weapon will do flame type Damage. This may not be applied to Claws or Unarmed strikes. Applying the oil takes 5 seconds. Example: “5 normal flame”.
```

Comment:
We do not have "Unarmed Strikes" anymore, unless RC is about to reintroduce duplicate abilities?  We could leave this as "may not be applied to Claws"  Same for below.